Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -273,23 +287,58 @@ void NetworkStorageManager::stopReceivingMessageFromConnection(IPC::Connection&
     });
 }
 
+#if PLATFORM(IOS_FAMILY)
+
+void NetworkStorageManager::excludeOriginFromBackupIfNecessary(OriginStorageManager& manager)
+{
+    auto now = WallTime::now();
+    if (auto lastCheckTimestamp = manager.lastExclusionCheckTimestamp()) {
+        if (m_backupExclusionPeriod > now - lastCheckTimestamp.value())
+            return;
+    }
```

Comment:
It looks like we're comparing against a time value in memory. Is that right? If so, we'll only include storage in backup if the browser runs continuously for 24 hours without exiting. On iOS jetsam makes that very unlikely. On macOS, whether it's likely or not depends on whether the user tends to quit apps.

I think we need some strategy to consult a value on disk instead. Perhaps we can consult the creation time of the directory.

Reply:
Oh, it looks like you do consult creation time lower down? Is the goal of this early return just to avoid doing overly frequent checks? If so, maybe it can use a smaller time value, like a few minutes.

Reply:
Even 30 minutes might be a long time on an iPhone. Maybe we can benchmark to discover the minimum delay we need? Hopefully we can get down to 5 minutes or 1 minute.

Reply:
Actually we may cache origin file creation time in memory, so we don't need the delay

Reply:
I've changed the patch to cache file creation time in memory, and compare now timestamp to it to decide if we need to exclude.